components/BWSHead/index.js

```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
+          <meta name='mobile-web-app-capable' content='yes' />
 
-          <meta name="referrer" content="origin-when-cross-origin" />
-          <meta
-            name="description"
-            content={pageDescription || defaultDescription}
-          />
-
-          <meta name="og:site_name" content={defaultPageTitle} />
-          <meta name="og:title" content={pageTitle || defaultPageTitle} />
-          <meta name="og:type" content={seoType || "website"} />
-          <meta name="og:image" content={pageImage || defaultImageUrl} />
-          <meta
-            name="og:description"
-            content={pageDescription || defaultDescription}
-          />
-          
-          <meta name="twitter:card" content="summary" />
-          <meta name="twitter:site" content="@dpla" />
-          <meta name="twitter:creator" content="@dpla" />
-          <meta name="twitter:image" content={pageImage || defaultImageUrl} />
+          <meta name='msapplication-TileColor' content='#4E4294' />
+          <meta name='msapplication-tap-highlight' content='no' />
+          <meta name='theme-color' content='#000000' />
+          <meta name='viewport' content='minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no, user-scalable=no, viewport-fit=cover' />
+                    
+          <link rel='apple-touch-icon' sizes='180x180' href='/static/favicons/apple-touch-icon.png' />
+          <link rel='icon' type='image/png' sizes='32x32' href='/static/favicons/favicon-32x32.png' />
+          <link rel='icon' type='image/png' sizes='16x16' href='/static/favicons/favicon-16x16.png' />
+          <link rel='manifest' href='/static/manifest.json' />
+          <link rel='mask-icon' href='/static/favicons/safari-pinned-tab.svg' color='#5bbad5' />
+          <link rel='shortcut icon' href='/static/favicons/favicon.ico' />
+              
+          <meta name='twitter:card' content='summary' />
+          <meta name='twitter:url' content='https://blackwomenssuffrage.dp.la' />
+          <meta name='twitter:title' content='BWS Collection' />
+          <meta name='twitter:description' content={pageDescription || defaultDescription}/>
+          <meta name='twitter:image' content={pageImage || defaultImageUrl}/>
           {pageImageCaption &&
-            <meta name="twitter:image:alt" content={pageImageCaption} />}
+            <meta name="twitter:image:alt" content={pageImageCaption} />} 
+          <meta name='twitter:creator' content='@dpla' />
+          <meta name='twitter:site' content='@dpla' />
+          <meta property='og:type' content={seoType || "website"} />
+          <meta property='og:title' content={pageTitle || defaultPageTitle} />
+          <meta property='og:description' content={pageDescription || defaultDescription}/>
+          <meta property='og:site_name' content={defaultPageTitle} />
+          <meta property='og:url' content='https://blackwomenssuffrage.dp.la' />
```

Comment:
Looks like this should be the URL of the page as well: https://neilpatel.com/blog/open-graph-meta-tags/